integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/cases/PluginOrderBugTest.kt

```diff
@@ -17,4 +18,14 @@ class PluginOrderBugTest {
         val buildResult = pluginOrderBugProject.cleanAndBuild()
         assertTrue(buildResult.isSuccessful, buildResult.output)
     }
+    
+    @Test
+    fun testResolvedCompilerPluginDependency() {
```

Comment:
I've added some check that there is `org.jetbrains.kotlin:atomicfu:1.9.20` dependency in the project classpath.

In the buildEnvironment we can also find the resolution string: 
`org.jetbrains.kotlin:atomicfu:1.6.20 -> 1.9.20`

but probably just checking for the kotlin version is enough

Reply:
Thanks to covering such a scenario!